src/test/java/com/training/sample/tests/Assignment2Test.java

```diff
@@ -0,0 +1,18 @@
+package com.training.sample.tests;
+
+import com.training.sample.pages.Assignment2;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class Assignment2Test {
+    Assignment2 assignment2 = new Assignment2();
+
+    @Test
+    public void testStringReplace() {
+        Assertions.assertEquals("Sam", assignment2.getStringReplace("Ram", "Sam"));
+        Assertions.assertEquals(
+                "Sachin Masterrrr",
+                assignment2.getStringReplace("Sachin Tendulkar", "Sachin Masterrrr"));
+        Assertions.assertEquals("Depp", assignment2.getStringReplace("Dell", "Depp"));
```

Comment:
For better maintenance, since `Depp` is used twice, lets assign that to a variable and then use the variable here.